todo/android/todo/src/main/java/com/mongodb/stitch/sdk/examples/todo/MainActivity.java

```diff
@@ -453,8 +464,20 @@ public void onComplete(@NonNull final Task<StitchUser> task) {
     private class MyUpdateListener implements ChangeEventListener<Document> {
         @Override
         public void onEvent(final BsonValue documentId, final ChangeEvent<Document> event) {
-            if (!event.hasUncommittedWrites()) {
-                // Add custom actions here
+
+            // Is this change coming from local or remote?
```

Comment:
[q] is this supposed to be part of this PR?

Reply:
ah, the comment makes it sound like a question we're unsure of. maybe say "This checks whether the change is coming from local or remote"